src/test/rustdoc/intra-doc-crate/submodule.rs

```diff
@@ -0,0 +1,14 @@
+// aux-build:submodule.rs
+// edition:2018
+extern crate bar as bar_;
```

Comment:
just use a different crate name here?

Reply:
The initial issue report said there was different behavior if the import worked in both crates, so I was trying to test that:
> Rustdoc correctly warns that it can't resolve the link, but this seems like a primary usecase of the feature to allow facade crates to have correctly linked docs when they rearrange types from underlying crates.

Reply:
Ah I see.